docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -5842,6 +5842,134 @@ Get details on a task
 - **404** – unknown task
 - **500** – server error
 
+## 3.11 Secrets
+
+**Note**: Secret operations require the engine to be part of a swarm.
+
+### List secrets
+
+`GET /secrets`
```

Comment:
we should prefix this now with the API version `/v1.25` (here, and other examples)